ANTLR/test/tests/Auxiliary/BaseTest.java

```diff
@@ -18,9 +18,8 @@
     protected CodePointCharStream createCharStream(String sourceCode) {
         try {
             return CharStreams.fromReader(new StringReader(sourceCode));
-        }
-        catch (Exception e) {
-            System.out.println(e);
+        } catch (Exception e) {
+            System.out.println(e.getMessage());
```

Comment:
Er dette stadig aktuelt eller er det kun debugging purpose?

Reply:
Stadig aktuelt. Vi gjorde det for, at vi ikke skulle have den der method signature på alle test funktionerne. Jeg har rettet det fra `e` til `e.getMessage()` fordi `e` ikke i sig selv har nogen betydning, når der skrives til skærmen.